app/src/main/res/layout/search_screen_item.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/frame_48"
+    android:layout_width="match_parent"
+    android:layout_height="@dimen/search_screen_item_height"
+    android:background="@drawable/search_screen_item_background"
+    android:layout_marginBottom="@dimen/search_screen_item_margin_bottom">
+
+<!--    Name of the city-->
+    <TextView
+        android:id="@+id/search_item_city_name"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/search_item_city_name_margin_top"
+        android:layout_marginBottom="@dimen/search_item_city_name_margin_bottom"
+        android:layout_marginStart="@dimen/search_item_city_name_margin_start"
+        android:textSize="@dimen/search_item_city_name_text_size"
+        android:text="Palermo"
+        android:textColor="@color/white"/>
+
+<!--    weather conditions precip-->
+    <TextView
+        android:id="@+id/search_item_weather_conditions"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignParentEnd="true"
+        android:layout_alignParentBottom="true"
+        android:layout_marginEnd="@dimen/search_item_wc_margin_end"
+        android:layout_marginBottom="@dimen/search_item_wc_margin_bottom"
+        android:text="precip"
+        android:textColor="@color/white"
+        android:textSize="@dimen/search_item_wc_text_size"/>
+
+<!--    degrees 12-->
+    <TextView
+        android:id="@+id/text"
```

Comment:
if you have search_item_something in the other layout element why this has an id like text??